src/vm/dllimportcallback.cpp

```diff
@@ -1416,6 +1416,7 @@ VOID UMThunkMarshInfo::RunTimeInit()
     {
         // caller pop
         m_cbRetPop = 0;
+        m_cbActualArgSize += numRegistersUsed * STACK_ELEM_SIZE;
```

Comment:
@janvorli Ok, thank you, I'll check that with more arguments. Have a nice vacation!